src/main/java/org/lightcouch/CouchDbProperties.java

```diff
@@ -192,4 +195,27 @@ public boolean isSSLAuthenticationDisabled() {
 		return disableSSLAuthentication;
 	}
 
+	/**
+	 * Returns the SSLSocketFactory that gets used when connecting to
+	 * CouchDB over a <code>https</code> URL.
+	 * @return An SSLSocketFactory, or <code>null</code>, which stands for
+	 *         the default SSLSocketFactory of the JRE.
+	 * @see #setSecureSSLSocketFactory(javax.net.ssl.SSLSocketFactory)
+	 */
+	public SSLSocketFactory getSecureSSLSocketFactory() {
+		return secureSSLSocketFactory;
+	}
+
+	/**
+	 * Specifies the SSLSocketFactory to use when connecting to CouchDB
+	 * over a <code>https</code> URL.
+	 * @param factory An SSLSocketFactory, or <code>null</code> for the
+	 *                default SSLSocketFactory of the JRE.
+	 * @see #getSecureSSLSocketFactory()
+	 */
+	public CouchDbProperties setSecureSSLSocketFactory(SSLSocketFactory factory) {
+		this.secureSSLSocketFactory = factory;
+		return this;
+	}
```

Comment:
Why are these called "setSecureSSLSocketFactory" and not just setSSLSocketFactory? I'm not sure the socket factory need be more secure...